### DISCUSSION TITLE
Need to be able to select the .row element of a grid.

### DISCUSSION THREAD

**Discussion by X3Technology** [vanjarosoftware/Vanjaro.Platform#1695] [General]
If you put a Grid block on the page, you can modify the properties of the .container element and the .col elements, but not the .row element.  If you select a column and press the 'select parent' button, it goes straight to the container element.  And the row is not selectable in the Navigator.

Since the .row element is already display:flex, a frequent use case is to use specify .align-items-center on the row which vertically centers shorter columns in the row.

**Comment by meetmandeep** [comment]
> If you put a Grid block on the page, you can modify the properties of the .container element and the .col elements, but not the .row element. 

Vanjaro UI is user-centric; not developer-centric. Therefore, the end goal is to make the user's life easier without worrying about any underlying code or understanding of framework. 

Can you share a use case or define the feature you're looking for? If you click on a Grid, you will find all the alignment options including flex options. 

![image](https://user-images.githubusercontent.com/29958123/131418737-ef3cb6c1-076f-4008-8284-897cc87a49e1.png)

@mathisjay This should've been a discussion. Please only create an issue with reporting a bug or requesting a feature. Most feature requests start as discussion first.

**Reply by X3Technology** [reply]
I was able to accomplish the same thing by selecting the column and using the Vertical Align option, which sets "align-self:center".   That works.
